tardis/visualization/widgets/shell_info.py

```diff
@@ -225,280 +212,202 @@ def __init__(self, hdf_fpath):
+            self.element_count_table.widths = {
+                "Element": 100,
+                f"Frac. Ab. (Shell {shell_num})": 140,
+            }
+            self._apply_tabulator_styles(self.element_count_table)
 
-        Returns
-        -------
-        ipywidgets.Box
-            Shell info widget containing all component widgets
-        """
-        if not is_notebook():
-            print("Please use a notebook to display the widget")
+            if not element_df.empty:
+                self.element_count_table.selection = [0]
+            else:
+                self.element_count_table.selection = []
+                self.update_ion_count_table(None)
         else:
-            # CSS properties of the layout of shell info tables container
-            tables_container_layout = dict(
-                display="flex",
-                align_items="flex-start",
-                justify_content="space-between",
-            )
-            tables_container_layout.update(layout_kwargs)
-
-            # Setting tables' widths
-            self.shells_table.layout.width = shells_table_width
-            self.element_count_table.layout.width = element_count_table_width
-            self.ion_count_table.layout.width = ion_count_table_width
-            self.level_count_table.layout.width = level_count_table_width
-
-            # Attach event listeners to table widgets
-            self.shells_table.on(
-                "selection_changed", self.update_element_count_table
-            )
-            self.element_count_table.on(
-                "selection_changed", self.update_ion_count_table
-            )
-            self.ion_count_table.on(
-                "selection_changed", self.update_level_count_table
-            )
-
-            # Putting all table widgets in a container styled with tables_container_layout
-            shell_info_tables_container = ipw.Box(
-                [
-                    self.shells_table,
-                    self.element_count_table,
-                    self.ion_count_table,
-                    self.level_count_table,
-                ],
-                layout=ipw.Layout(**tables_container_layout),
-            )
-            self.shells_table.change_selection([1])
-
-            # Notes text explaining how to interpret tables widgets' data
-            text = ipw.HTML(
-                "<b>Frac. Ab.</b> denotes <i>Fractional Abundances</i> (i.e all "
-                "values sum to 1)<br><b>W</b> denotes <i>Dilution Factor</i> and "
-                "<b>Rad. Temp.</b> is <i>Radiative Temperature (in K)</i>"
-            )
+            self.element_count_table.value = pd.DataFrame(columns=["Element", "Frac. Ab."])
+            self.element_title.object = "### Elements (No Shell Selected)"
+            self.element_count_table.selection = []
+            self.update_ion_count_table(None)
+
+    def update_ion_count_table(self, event):
+        """Update ion table based on element selection"""
```

Comment:
please look into the docstring format we use throughout tardis. You can look at the  tardis documentation for this

the tardis/base.py is a good example.